### DISCUSSION TITLE
maybe error in examples ?

### DISCUSSION THREAD

**Discussion by smoegel** [alitto/pond#76] [Q&A] [answered]
// Create a pool with limited concurrency
pool := pond.NewPool(10)

// Create a task group
**group := pool.Group()  <----**
//group:=pool.NewGroup()

// Submit a group of tasks
for i := 0; i < 20; i++ {
	i := i
	group.Submit(func() {
		fmt.Printf("Running group task #%d\n", i)
	})
}

// Wait for all tasks in the group to complete
err := group.Wait()
if err != nil {
	fmt.Printf("Failed to complete group tasks: %v", err)
} else {
	fmt.Println("Successfully completed all group tasks")
}

**Comment by alitto** [comment, marked as answer]
Hey!

Thanks for pointing this out :slightly_smiling_face:, i just pushed a commit to fix that.

Cheers
